src/Dfe.PlanTech.Web/Program.cs

```diff
@@ -78,4 +78,9 @@
     name: "default",
     pattern: "{controller=Pages}/{action=GetByRoute}/{id?}");
 
+app.MapControllerRoute(
```

Comment:
Move this before the default.  The default should be at the end, so if it can't find a match it goes to that.

Reply:
Fixed.